text.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo">Jo</div>
+            <div></div>
+            <div class="menu">
+                <a href="#" class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </a>
```

Comment:
- [ ] Kindly note that your hamburger will represent the navigation links of your website when you eventually add it, so to make your website semantic, it will be better if you wrap your hamburger in a `<nav>` tag instead of `<a href="#" class=" nav">` so that screen readers and search engines will understand that the particular section is for navigations 👍🏽